README.md

```diff
@@ -252,29 +252,29 @@ email = "martinvonz@google.com"
 ## Command-line completion
 
 To set up command-line completion, source the output of
```

Comment:
Optional: perhaps temporarily mention how this is called `debug` in the released 0.7 version?

Reply:
Good idea!